d3a_api_client/utils.py

```diff
@@ -264,3 +264,25 @@ def log_bid_offer_confirmation(message):
                          f"{round(energy, 2)} kWh at {price} cts/kWh")
     except Exception as e:
         logging.error(f"Logging bid/offer info failed.{e}")
+
+
+def flatten_info_dict(indict: dict) -> dict:
+    """
+    wrapper for _flatten_info_dict
+    """
+    if indict == {}:
+        return {}
+    outdict = {}
+    _flatten_info_dict(indict, outdict)
+    return outdict
+
+
+def _flatten_info_dict(indict: dict, outdict: dict):
+    """
+    Flattens market_info/tick_info information trees
+    outdict will hold references to all area subdicts of indict
+    """
+    for area_name, area_dict in indict.items():
+        outdict[area_name] = area_dict
```

Comment:
As discussed, in general I agree with the flattening process, however using names as a key here can be potentially dangerous, hence the research team needs to be aware of potential name collisions. In general I would feel far safer if the users used uuids to call these functions, however it depends on the research team

Reply:
You are right. I agree. I wanted to discuss this (but got interrupted and forgot then).
Let me ask the research team for their opinion.